bin/tasks/gitInfo.js

```diff
@@ -21,8 +21,10 @@ const TesterSkipBuildMutation = `
 export const setGitInfo = async (ctx, task) => {
   const { branchName, patchBaseRef, fromCI: ci } = ctx.options;
   ctx.git = await getCommitAndBranch({ branchName, patchBaseRef, ci, log: ctx.log });
-  ctx.git.slug = ctx.git.slug || (await getSlug());
   ctx.git.version = await getVersion();
+  if (!ctx.git.slug) {
+    ctx.git.slug = await getSlug().catch((e) => ctx.log.warn('Failed to retrieve slug', e));
+  }
 
   if (ctx.options.ownerName) {
     ctx.git.slug = ctx.git.slug.replace(/[^/]+/, ctx.options.ownerName);
```

Comment:
Will this code fail without the slug set?

Reply:
referring to:
```
 ctx.git.slug = ctx.git.slug.replace(/[^/]+/, ctx.options.ownerName);
 ```

Reply:
Good catch.